pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java

```diff
@@ -2541,7 +2541,25 @@ public boolean isWrapperFor(Class<?> iface) throws SQLException {
 
   public ResultSet getFunctions(String catalog, String schemaPattern, String functionNamePattern)
       throws SQLException {
-    return getProcedures(catalog, schemaPattern, functionNamePattern);
+    String sql;
+    sql = "SELECT NULL AS FUNCTION_CAT, n.nspname AS FUNCTION_SCHEM, p.proname AS FUNCTION_NAME, "
+            + "d.description AS REMARKS, "
+            + DatabaseMetaData.procedureReturnsResult + " AS FUNCTION_TYPE, "
```

Comment:
Hi @jorsol, hum... I actually just took the original query from `getProcedures` and fix the column names and removed extra reserved fields. I didn't check this "FUNCTION_TYPE" is actually valid or not, since I assume it was working before. Let me take a look again...